configs/xcache/config.d/04-debug-tuning.cfg

```diff
@@ -0,0 +1,16 @@
+#
+# Configuration for debugging an XCache server
+#
+# ************************************************************
+# * NOTE: Changes to this file will be PRESERVED on upgrades *
+# ************************************************************
+#
+# This file is part of the XCache Daemon
+
+# Uncomment the following configuration to assist with debugging:
+
+
+#xrd.trace all
+#ofs.trace all
+#http.trace all
+#cms.trace all
```

Comment:
I forgot that we already have a logging config file for XCache. Let's just add these commented lines to the end of https://github.com/opensciencegrid/xcache/blob/master/configs/xcache/config.d/90-xcache-logging.cfg

Reply:
Yea removed it and moved things to 90